rules/namespace.go

```diff
@@ -116,17 +116,17 @@ func newNamespace(namespace *rulepb.Namespace) (Namespace, error) {
 }
 
 // NamespaceView returns the view representation of a namespace object.
-func (n Namespace) NamespaceView(snapshotIdx int) (*models.NamespaceView, error) {
+func (n Namespace) NamespaceView(snapshotIdx int) (models.Namespace, error) {
```

Comment:
perhaps change this function name to `NamespaceModel`.

Reply:
I have gotten a little further in the diff it seems like at least we are consistent here, and while the function name longer no longer directly matches the struct name that it returns it is descriptive of what the function returns and can be left alone.

Reply:
Perhaps we should rename the `models` package to `view`? From a MVC perspective, these data structures are for presentation purposes and as such are views rather than models (which are the internal data structure defined in the `rules` package).